docs/requirements.rst

```diff
@@ -9,38 +9,59 @@ fulfill these aspects.
+the output written by the workflow system and the running processes. However, ideally, the workflow system allows for submission of the workflow in
+the form of a process running in the background, while still providing means to monitor the state of the execution.
+
+Evaluation criteria:
+
+1. Only way to monitor the workflow is to watch the console output
+2. The workflow system provides a way to query the execution status at any time
 
 .. _provenance:
+Data provenance
+---------------
+The data provenance graph contains, for a particular execution of the workflow, which data and processes participated in the generation of a particular
+piece of data. Thus, this is closely related to the workflow itself, which can be thought of as a template for how that data generation should take place.
+However, a concrete realization of the workflow must contain information on the exact input data and parameters that were used, possibly along with meta
+information on the person that executed the workflow, the compute ressources used and the time it took to finish. Collection of all relevant information,
+its storage in machine-readable formats and subsequent publication alongside the data can be very useful for future researchers in order to understand
+how exactly the data was produced. Ideally, the workflow system has the means to automatically collect this information upon workflow execution.
 
-Data provenance graph
----------------------
-After the successful execution of a workflow, it should be possible to obtain
-the provenance graph for all produced or modified data within the workflow.
+Evaluation criteria:
 
+1. The workflow system provides no means to export relevant information from a particular execution
+2. Upon workflow execution, the tool writes metadata files alongside the results, overwriting them upon re-execution
+3. Produced data is stored in a database, allowing to uniquely associate produced data with particular workflow instantiations
 
-.. _requirements_metadata:
 
-Metadata
+.. _requirements_metadata:
+Metadata (old?)
 --------
-To make a published workflow compliant with the
-`FAIR principles <https://www.go-fair.org/fair-principles/>`_, appropriate metadata
-about the workflow has to be provided. Workflow tools may provide the possibility
-to export detailed information on the used software components, for instance, the
-exact versions, the chosen parameters, etc. This metadata should be exportable into
+To make a published workflow compliant with the `FAIR principles <https://www
+.go-fair.org/fair-principles/>`_, appropriate metadata about the workflow has
+to be provided. Workflow tools may provide the possibility to export detailed
+information on the used software components, for instance, the exact
+versions, the chosen parameters, etc. This metadata should be exportable into
 widely-used data formats such as JSON.
```

Comment:
I think there is an overlap with provenance here (at least from the perspective of the provenance graph that is exported by some of the tools). For example, versions of the code used are part of the provenance graph in AiiDA. 
Maybe we could make that more explicit, but from my point of view level two of the provenance requirement covers this.